add csw shortcut

tam5 opened the pull request:
closes #147.

ninrod commented:
I wrote two commits after your PR but you should be able to rebase with no problems.

tam5 commented:
I've rebased without a problem.

I've also added `W`, so now `w` and `W` will work.

I'm having some trouble with `s` and `p`, they seem to work most of the time, but sometimes throw an error that is not thrown when using `ysi`. Still looking into it

ninrod commented:
Looks good to me. Merging. Thanks for contributing.
